packages/builder/src/internal/core/config/config-validation.ts

```diff
@@ -102,31 +95,23 @@ const HDAccountsConfig = t.type({
   path: optional(t.string),
 });
 
-const OtherAccountsConfig = t.type({
-  type: t.string,
-});
-
-const NetworkConfigAccounts = t.union([
+const NetworkAccounts = t.union([
   t.literal("remote"),
   t.array(t.string),
   HDAccountsConfig,
-  OtherAccountsConfig,
 ]);
 
 const HttpHeaders = t.record(t.string, t.string, "httpHeaders");
 
 const HttpNetworkConfig = t.type({
-  chainId: optional(t.number),
+  chainName: optional(t.number),
   from: optional(t.string),
-  gas: optional(t.union([t.literal("auto"), t.number])),
-  gasPrice: optional(t.union([t.literal("auto"), t.number])),
-  gasMultiplier: optional(t.number),
   url: optional(t.string),
-  accounts: optional(NetworkConfigAccounts),
+  accounts: optional(NetworkAccounts),
   httpHeaders: optional(HttpHeaders),
 });
 
-const NetworkConfig = t.union([BuilderNetworkConfig, HttpNetworkConfig]);
+const NetworkConfig = t.union([AlgoDevChainConfig, HttpNetworkConfig]);
```

Comment:
How is `AlgoDevChainConfig` different from non `*Dev*` config? Previously it was more general.

Reply:
BuilderNetwork is a config for a internal chain implementation. We also have this as an extra milestone at the end of the project. So it will require different arguments. It's not a `devnet`

Reply:
At the end of the project -- so currently it's not needed to have `Dev` in it's name. It could be `AlgorandNetworkConfig` or name shouldn't change.

Reply:
The Dev means that it's a not a normal `algorand` node. But  a stripped node. If you find a better name then let me know on telegram and we can rename it